### PR TITLE
Update with-tyk-on-premises.md

### PR DESCRIPTION
**Pull request by sedkis**
remove old reference to `tyk multi-cloud` and make the differences more clear

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/manage-multiple-environments/with-tyk-on-premises.md [hunk 1]
```diff
@@ -77,6 +77,6 @@ This will also set the tags for the API and when API requests are made through t
...
+API Sharding with Self-Managed is very flexible, but it behaves a little differently to sharding with Tyk Cloud Hybrid & Tyk Global Self-Managed deployments. The key difference is that with the latter, you can have federated Gateway deployments with **their own redis databases**.  However with Tyk Self-Managed the zoning is limited to tags only, and must share a single Redis database.
 
-To isolate Self-Managed installations across data centers you will need to use our Multi Data Center Bridge component. This system powers the functionality of Tyk Multi-Cloud in our cloud and is available to our Enterprise customers as an add-on.
+To isolate Self-Managed Gateway installations across data centers you will need to use Tyk Multi Data Center Bridge component. This system powers the functionality of Tyk Cloud & Tyk Cloud Hybrid in our cloud and is available to our Enterprise customers as an add-on.
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/manage-multiple-environments/with-tyk-on-premises.md]:
```suggestion
To isolate Self-Managed Gateway installations across data centers you will need to use Tyk Multi Data Center Bridge component. This system powers the functionality of Tyk Cloud & Tyk Cloud Hybrid in our cloud and is available to our enterprise customers as an add-on.
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
lgtm, minor suggestion change

**Review by dcs3spp — APPROVED** [review]

File: tyk-docs/content/advanced-configuration/manage-multiple-environments/with-tyk-on-premises.md [hunk 2]
```diff
@@ -77,6 +77,6 @@ This will also set the tags for the API and when API requests are made through t
...
+API Sharding with Self-Managed is very flexible, but it behaves a little differently to sharding with Tyk Cloud Hybrid & Tyk Global Self-Managed deployments. The key difference is that with the latter, you can have federated Gateway deployments with **their own redis databases**.  However with Tyk Self-Managed the zoning is limited to tags only, and must share a single Redis database.
 
-To isolate Self-Managed installations across data centers you will need to use our Multi Data Center Bridge component. This system powers the functionality of Tyk Multi-Cloud in our cloud and is available to our Enterprise customers as an add-on.
+To isolate Self-Managed Gateway installations across data centers you will need to use Tyk Multi Data Center Bridge component. This system powers the functionality of Tyk Cloud & Tyk Cloud Hybrid in our cloud and is available to our enterprise customers as an add-on.
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/manage-multiple-environments/with-tyk-on-premises.md:82]: @dcs3spp @sedkis  could be great to add here a CTA button to contact us (in a new pr :) )

**Pull request merged**